src/lib.rs

```diff
@@ -68,7 +69,13 @@ impl Server {
             return;
         }
 
-        let response = (self.handler)(request, response_builder);
+        let response = (self.handler)(request, response_builder).unwrap_or_else(|_| {
```

Comment:
i'd like this error in a separate file if we could.

Reply:
maybe a `status` helper?

Reply:
I'm not sure what this means.

Reply:
i want to get the html out of this, basically. i can write something up later.